builtins/safe-default-configuration.json

```diff
@@ -762,6 +1172,258 @@
     {
       "name": "scriptlevel",
       "namespace": null
+    },
+    {
+      "name": "space",
+      "namespace": "http://www.w3.org/XML/1998/namespace"
```

Comment:
If this is the only namespaced attribute we end up allowing let's scrap it as it's useless anyway.

Reply:
It's marked as deprecated on [MDN](https://developer.mozilla.org/en-US/docs/Web/SVG/Reference/Attribute/xml:space) as well.

Reply:
👍,  as above :)

Reply:
Done.